cmd/earthly/secret_cmds.go

```diff
@@ -291,7 +291,12 @@ func (app *earthlyApp) actionSecretsListV2(cliCtx *cli.Context) error {
 	}
 
 	for _, secret := range secrets {
-		fmt.Println(secret.Path)
+		display := secret.Path
+		if !strings.HasPrefix(display, "/user") {
+			prefix := fmt.Sprintf("/%s/%s/", cliCtx.String("org"), cliCtx.String("project"))
+			display = strings.Replace(display, prefix, "", 1)
```

Comment:
Wondering if this can be brittle to future changes? This replacement doesn't actually require for the "prefix" to be at the beginning, yet it yanks it regardless.

What do you think about using `TrimPrefix` instead?

Reply:
Good call. I forgot about that function.